src/main.rs

```diff
@@ -21,6 +22,14 @@ impl fmt::Display for Config {
     }
 }
 
+fn get_user_input() -> String {
```

Comment:
@sabjorn maybe we should just use [rprompt](https://docs.rs/rprompt/latest/rprompt/#)?

Reply:
I think for now that is good!

Reply:
Maybe I'll refactor later.